content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Installing External Istiod [experimental]
+description: Install external Istiod and remote config cluster.
```

Comment:
Just drop `config cluster` from the description for now ... we don't really document that yet.

Reply:
Done